docs/CODING.md

```diff
@@ -73,3 +73,23 @@ and reviewers. This document lists such coding conventions.
   few reasons to use asserts).
 * Any conversion between strings and bytes happens with `os.fsencode/fsdecode`.
 * Messages are to be expressed in strings (log, errors, etc).
+
+## DOCSTRINGS
+
+* docstrings are quoted with three double quotes (see above).
+* each triple quotes are alone on their line (this is a deviation from some
+  recommendations) so that it is readable.
+* each docstring starts with a single line summarizing the purpose of the
+  object.
+* further descriptions are written after a blank line.
+
+## IMPORT
```

Comment:
Same here we should consider adding flake8

Reply:
See above and #530